torch/testing/_internal/common_modules.py

```diff
@@ -318,6 +319,19 @@ def no_batch_dim_reference_mha(m, p, *args, **kwargs):
         return (output[0].squeeze(batch_dim), output[1].squeeze(0))
 
 
+def no_batch_dim_reference_lstmcell(m, p, *args, **kwargs):
```

Comment:
I thought of updating the generic, `no_batch_dim_reference_fn`. But it didn't seem clear if we should also add multi-output support since it also supports criterion which are always single output fn. Writing this seemed trivial so went ahead with adding this one.

Reply:
It's a bit tricky to have the generic one handle criteria & multi-output at once, right? If so, I'm okay with just doing this

Reply:
Let's just do this for now :)